example/cmd/device-simple/res/configuration.toml

```diff
@@ -41,11 +42,12 @@ Type = 'consul'
   Port = 48081
 
 [MessageQueue]
-Enabled = false
-Protocol = 'tcp'
+Protocol = 'redis'
 Host = 'localhost'
-Port = 1883
-Type = 'mqtt'
+Port = 6379
+Type = 'redisstreams'
```

Comment:
are we still calling it 'redisstreams' when we are using redis pub/sub?  Are they the same?
I feel confused about the message type name.  How about just calling it 'redis'?

Reply:
Done